adagrams/game.py

```diff
@@ -1,11 +1,113 @@
+import random
+
+DISTRIBUTION_OF_LETTERS = {
+    "A": 9,
+    "B": 2,
+    "C": 2,
+    "D": 4,
+    "E": 12,
+    "F": 2,
+    "G": 3,
+    "H": 2,
+    "I": 9,
+    "J": 1,
+    "K": 1,
+    "L": 4,
+    "M": 2,
+    "N": 6,
+    "O": 8,
+    "P": 2,
+    "Q": 1,
+    "R": 6,
+    "S": 4,
+    "T": 6,
+    "U": 4,
+    "V": 2,
+    "W": 2,
+    "X": 1,
+    "Y": 2,
+    "Z": 1,
+}
+
+SCORE_CHART = {
+    "A": 1,
+    "B": 3,
+    "C": 3,
+    "D": 2,
+    "E": 1,
+    "F": 4,
+    "G": 2,
+    "H": 4,
+    "I": 1,
+    "J": 8,
+    "K": 5,
+    "L": 1,
+    "M": 3,
+    "N": 1,
+    "O": 1,
+    "P": 3,
+    "Q": 10,
+    "R": 1,
+    "S": 1,
+    "T": 1,
+    "U": 1,
+    "V": 4,
+    "W": 4,
+    "X": 8,
+    "Y": 4,
+    "Z": 10,
+}
+
+
 def draw_letters():
-    pass
+    letters_drawn = []
+    available_letters = []
+    for letter, limit in DISTRIBUTION_OF_LETTERS.items():
+        available_letters.append(letter)
+    while len(letters_drawn) <= 9:
+        letter_choice = random.choice(available_letters)
+        letter_count = letters_drawn.count(letter_choice)
```

Comment:
When using built-in functions like `count`, you'll want to consider their time complexity. What's the big-O complexity of this built-in function? (You can search for it on stack overflow, but you can logically conclude a lower bound of its time complexity by thinking about what it *must* do in order to function correctly.) Given the big-O of `count`, what does it do to the big-O of your overall function? Think about the function as-is, then also consider what might happen if requirements changed, for example, if we were required to draw an arbitrary number of letters.

Also, after you think about that, it's okay to note that analysis in a comment and then conclude that the performance of `count` is just fine for this implementation given the current, real-world constraints. 😄